src/render/mod.rs

```diff
@@ -242,6 +244,9 @@ impl Plugin for TilemapRenderingPlugin {
             Shader::from_wgsl
         );
 
+        app.init_resource::<ModifiedImageHandles>()
```

Comment:
Should these main-world resources / systems be in in `build` rather than `finsih`?

Reply:
Yep! I didn't even realize this was in `finish` haha. Done.